cache

```diff
@@ -414,12 +414,15 @@ cache::lftp_put() {
     if cache::file_is_present ${SEMAPHORE_CACHE_KEY}; then
       cache::log "Key '${SEMAPHORE_CACHE_KEY}' already exists."
     else
+      archive_start=$(date +'%s')
```

Comment:
Does seconds give us enough precision here? Though as I `man`ed through `date` it seems to support only `nanoseconds` and `seconds` precision.

Reply:
@skipi 
The reason behind displaying this timing is to know if creating the archive or the actual upload is taking too long (if that's the case). Basically, we want to know if it is 10s or 500s, when comparing to previous builds. In my opinion, nanoseconds don't matter here.